webapp/src/widgets/valueSelector.tsx

```diff
@@ -114,6 +114,10 @@ const valueSelectorStyle = {
         padding: '0 8px',
         overflow: 'unset',
     }),
+    placeholder: (provided: CSSObject): CSSObject => ({
+        ...provided,
+        color: 'rgba(var(--center-channel-color-rgb), 0.4)',
+    }),
```

Comment:
May be this style for placeholder should be moved to [getSelectBaseStyle](https://github.com/mattermost/focalboard/blob/main/webapp/src/theme.ts#L242)

Reply:
If we move it to `getSelectBaseStyle` we have to specify select styling for places other than card properties. Either way, we'll end up having something specified in base and it being overridden in uses, so I think what we have here is fine.